docs/download.ipynb

```diff
@@ -12,23 +12,163 @@
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
     "stac_client = Client.open(\"https://csdap.earthdata.nasa.gov/stac/\")\n",
-    "item_search = stac_client.search(collections=[\"planet\"], intersects=LONGMONT)\n",
+    "rows = []\n",
+    "for collection in stac_client.collection_search().collections_as_dicts():\n",
+    "    rows.append([collection[\"id\"], collection.get(\"title\")])\n",
+    "\n",
+    "tabulate.tabulate(rows, headers=[\"id\", \"title\"], tablefmt=\"html\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "deea059e",
+   "metadata": {},
+   "source": [
+    "## Finding Planet data\n",
+    "\n",
+    "One of the CSDA vendors, [Planet](https://www.planet.com/), collects imagery that is easy to visualize and interpret, so let's use their data for this notebook.\n",
+    "We can use a [STAC API search query](https://github.com/radiantearth/stac-api-spec/tree/release/v1.0.0/item-search#query-parameter-table) to narrow our request.\n",
+    "In this example, we'll find [STAC items](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md) that:\n",
+    "\n",
+    "- Have a cloud cover of less than or equal to 25%\n",
+    "- Were collected in 2021\n",
+    "- Intersect [Longmont, CO, USA](https://en.wikipedia.org/wiki/Longmont,_Colorado)\n",
+    "\n",
+    "### What's the deal with the `cql2` stuff?\n",
+    "\n",
+    "[cql2](https://www.ogc.org/standards/cql2/) is an [Open Geospatial Consortium](https://www.ogc.org/) standard for querying spatial data.\n",
+    "Our STAC API only understands `cql2-json`, but `cql2-json` can be a little awkward to write.\n",
+    "`cql2-text` is much more intuitive, so we write that and then convert it to `cql2-json` using [this helper package](https://github.com/developmentseed/cql2-rs/).\n",
+    "This should become automatic via **pystac-client** [soon](https://github.com/stac-utils/pystac-client/issues/803)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "1",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "18\n"
+     ]
+    }
+   ],
+   "source": [
+    "item_search = stac_client.search(\n",
+    "    collections=[\"planet\"],\n",
+    "    intersects=LONGMONT_CO_USA,\n",
+    "    filter=cql2.parse_text(\"eo:cloud_cover<25\").to_json(),\n",
+    "    datetime=\"2021\",\n",
```

Comment:
It may also be useful to expand the example datetime search for someone who may not be familiar. Perhaps even showing the range search?
```suggestion
    "    datetime=\"2021-03-01/2021-05-31\",\n",
```

Reply:
Interesting - the date range as string text I suggested is what seemed to work for me locally (ie, it returned 5 items.. 🤔) but I didn't look that closely to verify their dates.

That said, I think the note you mentioned about other possibilities would also work (for me at least).